nestkernel/connector_base.h

```diff
@@ -98,6 +129,7 @@ class ConnectorBase
 
   virtual size_t get_num_connections() = 0;
   virtual size_t get_num_connections( synindex syn_id ) = 0;
+  virtual size_t get_num_connections( size_t target_gid, size_t thrd, synindex syn_id ) = 0;
```

Comment:
Hi dear Tammo, thanks for taking the time to check the code and for your very useful comments. This function is used by the structural plasticity manager. I know it is not visible right now, because the structural plasticity manager class will be part of the third pull request. The whole implementation of structural plasticity was split into three pieces in order to make easier the code review process. I tried to make each piece self explanatory and independent from each other but in this case the usage is not clear. Sorry for the confusion.